### PR TITLE
Hide scroll bar in iframe

### PR DESCRIPTION
**Pull request by ocallesp**
https://github.com/dotnet/try/issues/1198

hideScrollBar

Thanks @danzhu54  for the fix suggestion

### CODE REVIEW

File: src/microsoft-trydotnet-editor/src/index.css [hunk 1]
```diff
@@ -2,6 +2,7 @@ body {
 	width: auto;
 	height: auto;
 	margin: 0;
+	overflow: hidden;
```
Comment (jonsequitur) [inline src/microsoft-trydotnet-editor/src/index.css:5]: If the content is sized correctly, I would think we shouldn't need to do this.

Comment (ocallesp) [reply src/microsoft-trydotnet-editor/src/index.css:5]: It seems that the issue is not in the content.
Here is the animated gif showing the behavior WITHOUT 'overflow: hidden'
ScrollBar

**Comment by ocallesp** [comment]
@jwmunn does this change break something on Learning ?

File: src/microsoft-trydotnet-editor/src/index.css (hunk 1 above)
Comment (jonsequitur) [reply src/microsoft-trydotnet-editor/src/index.css:5]: Let's find some time to discuss.

**Comment by jonsequitur** [comment]
We should close this PR and take a different approach. The editor's API can be called directly by the host page and that's going to be more flexible than one-size-fits-all attempts to remedy this with CSS.

**Comment by danzhu54** [comment]
@jonsequitur will you expose a setSize function on your API instead? I believe this change will resolve all editor overflow issues. Learn is facing the same issue:
image

**Comment by colombod** [comment]
Who is the owner of the scroll bars? is it the iframe or the div inside?

**Comment by colombod** [comment]
This is what I see:
The content of the iframe is bigger than the iframe:
image
the element with class monaco-editor is se to this the size 1245px X 839px in the style sheet (side effect of calling the layout code)

this is the iframe size : 
image

and this is the actual size of the div 
image

**Comment by jonsequitur** [comment]
Let's move this discussion to the issue.

https://github.com/dotnet/try/issues/1198#issuecomment-2148221113

**Pull request closed without merging**